frontend/packages/client/src/components/CommunityLinks.js

```diff
@@ -10,12 +11,7 @@ export default function CommunityLinks({
 } = {}) {
   return (
     <div className="columns my-0 is-multiline">
-      <div
-        className="column is-12 has-text-weight-bold is-uppercase small-text"
-        style={{ marginBottom: '20px' }}
-      >
-        Links
-      </div>
+      <Title>Links</Title>
```

Comment:
this one need classes `column is-12` should Title allow classNames prop like `classNames={"column is-12"}`
<img width="396" alt="Screen Shot 2022-07-14 at 13 58 47" src="https://user-images.githubusercontent.com/7526740/179039340-8aed6704-7943-4b9b-9aee-c2f60125dbe8.png">